repl_mon.c

```diff
@@ -142,7 +145,11 @@ repl_mon_update_data()
     appendStringInfo(&buf, "WITH repl AS ("
             "SELECT count(*) AS cnt FROM pg_catalog.pg_stat_replication "
             "WHERE state='streaming') UPDATE public.%s "
+#if PG_VERSION_NUM < 100000
```

Comment:
It would be better to define a variable (i.e. `wal_function`) earlier (and make it different for different versions) and format query text with this variable.